apps/condo/domains/common/utils/helpers.ts

```diff
@@ -47,8 +47,6 @@ type ObjectWithAddressInfo = Pick<Property, 'address' | 'addressMeta'>
  * @param property
  */
 export const getAddressDetails = (property: ObjectWithAddressInfo) => {
-    if (isEmpty(property)) return
```

Comment:
Is it OK that some address parts may be "undefined"?
For example, `const streetWithType = get(addressMeta, 'street_with_type')` is used in concatenated strings.
<img width="229" alt="image" src="https://github.com/open-condo-software/condo/assets/25384290/44e12b2d-65b7-4fd1-ac08-3c9c6d032bf6">